events/famnm-forums.html

```diff
@@ -16,16 +31,35 @@
       <h1 class="page-header">2018 FAMNM Forums</h1>
       <p class="text-par font-weight-bold">FAMNM is once again hosting the FAMNM Forums on U of M's North Campus on November 17! Check back soon for details!</p>
       <p class="text-center"><a href="https://goo.gl/forms/BhazwWADHde8aEKY2" class="btn btn-primary" onclick="openInModal(event, '2018 FAMNM Forums Team Registration')">Click here to register</a></p>
+      <h2>Map</h2>
+      <div id="map-wrap">
+          <div id="map"></div>
+      </div>
+      <div id="legend" class="row font-weight-bold">
+          <span class="col-sm">
+              <img src="../img/kickoff/icons/parking.png" alt="Parking"/>
```

Comment:
![image](https://user-images.githubusercontent.com/8966465/48454195-5bfff300-e784-11e8-8baf-8a5ce341449a.png)
the parking icon is hard to see in satellite mode of the map, is it possible for this to be made red/light blue/color that stands out?